### PR TITLE
Changed return MinimizedWorkEntity to WorkEntity, Updated Mapper

### PR DESCRIPTION
**Pull request by nitw-shobhit**
https://trello.com/c/KJAqTh4x/2163-v2-0-verify-all-activities-in-the-summary-have-a-created-and-modified-date

### CODE REVIEW

File: orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java [hunk 1]
```diff
@@ -222,8 +222,8 @@ private void setIncomingWorkPrivacy(WorkEntity workEntity, ProfileEntity profile
     @Override
     @Cacheable(value = "works-summaries", key = "#orcid.concat('-').concat(#lastModified)")
     public List<WorkSummary> getWorksSummaryList(String orcid, long lastModified) {
-        List<MinimizedWorkEntity> works = workDao.findWorks(orcid);        
-        return jpaJaxbWorkAdapter.toWorkSummaryFromMinimized(works);
```
Comment (rcpeters) [inline orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java]: Will this start forcing loading the full work?

Comment (nitw-shobhit) [inline orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java]: @rcpeters yup.. I removed MinimizedWorkEntity and put WorkEntity. MinimizedWorkEntity has no createdDate or lastModified fields

Comment (rcpeters) [inline orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java]: Add them. There are some records that would crash the system if we load the all their complete works.

Comment (nitw-shobhit) [inline orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java]: @rcpeters It has those fields. I missed the extended BaseEntity. Reverted and updated just the Mapper.

**Pull request merged**